modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- New `editor.undoManager.fireIfChanged` API to fire change event only if the last undoManager content layer is diffrent from the current editor content #TINY-8641
```

Comment:
```suggestion
- New `dispatchIfChanged` UndoManager API to fire the `change` event if the last UndoManager content layer is different from the current editor content #TINY-8641
```
This still mentioned `fireIfChanged`. I've also reworded it a little.

Reply:
done